rosco-web/config/packer/aws-ebs.json

```diff
@@ -15,6 +15,7 @@
     "build_host": "",
     "deb_repo": "",
     "packages": "",
+    "upgrade": "",
```

Comment:
Please make the same changes to `gce.json` since it also employs `install_packages.sh`.
Trying to keep the behavior of the two templates consistent where reasonable to do so.